src/app/pages/data-protection/cloudsync/cloudsync-form/cloudsync-form.component.ts

```diff
@@ -407,6 +419,26 @@ export class CloudsyncFormComponent {
         }
       }
     });
+
+    this.form.controls.transfers.valueChanges.pipe(untilDestroyed(this)).subscribe((value: number) => {
+      const transfersDefaultValues = this.transfersDefaultOptions.map((transfers) => transfers.value);
```

Comment:
This method hard to read. Can we split it and maybe merge with code below for showing Custom option.